docs/source/contributor-guide/development.md

```diff
@@ -91,13 +91,15 @@ Comet has a plan stability testing framework that can be used to test the stabil
 The plan stability testing framework is located in the `spark` module and can be run using the following command:
 
 ```sh
-./mvnw -pl spark -Dsuites="org.apache.spark.sql.comet.CometTPCDSV1_4_PlanStabilitySuite" test
+./mvnw -pl spark -Dsuites="org.apache.spark.sql.comet.CometTPCDSV1_4_PlanStabilitySuite" -nsu test
+./mvnw -pl spark -Dsuites="org.apache.spark.sql.comet.CometTPCDSV1_4_PlanStabilitySuite" -Pspark-3.5 -nsu test
```

Comment:
Since you added these steps to the testing commands, does it makes sense to duplicate them below for the generation commands?

Reply:
Thanks, updated.